src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = "white";
+    private static final int FIGURE_COUNT = 5;
+    private static final int MAX_VALUE = 10;
+
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(FIGURE_COUNT);
+        switch (index) {
+            case 0:
+                return new Circle(colorSupplier.getRandomColor(), random.nextInt(MAX_VALUE));
+            case 1:
+                return new Rectangle(colorSupplier.getRandomColor(), random.nextInt(MAX_VALUE),
+                        random.nextInt(MAX_VALUE));
```

Comment:
Consider creating utility method like "getRandomNumber()", so as not to duplicate this "random.nextInt(..)"